orttraining/orttraining/training_ops/rocm/tensor/gather_grad_impl.cu

```diff
@@ -161,24 +159,20 @@ void GatherGradImpl(
       original_indices.get(), original_indices_sorted.get(),
       num_indices));
 
+  dim3 block(GPU_WARP_SIZE, 4);
+  dim3 grid(CeilDiv(num_indices, 4), CeilDiv(stride, GridDim::maxElementsPerThread * GPU_WARP_SIZE));
```

Comment:
Previously it was `2*ceil(stride/128)` and now it's `ceil(stride/256)`?

Reply:
I changed the block size as well. Previously, it was block(GPU_WARP_SIZE/4, 4), which is a bit weird.

Reply:
Oh, I see. I guess the perf is unaffected then?

Reply:
yes, I measured and it is same.